src/main/java/uk/gov/hmcts/darts/cases/service/impl/CaseServiceImpl.java

```diff
@@ -71,8 +73,12 @@ public List<ScheduledCase> getHearings(GetCasesRequest request) {
             request.getDate()
         );
         createCourtroomIfMissing(hearings, request);
-        return casesMapper.mapToScheduledCases(hearings);
 
+        List<ScheduledCase> scheduledCases = casesMapper.mapToScheduledCases(hearings);
+
+        logApi.casesRequestedByDarPc(request);
```

Comment:
Maybe log at the start, in case an exception happens during the mapping?

Reply:
Yeah, was thinking about this wrong